config/default.yaml

```diff
@@ -211,6 +211,13 @@ protections:
 roomStateBackingStore:
   enabled: false
 
+#safeMode:
+#  # The option for entering safe mode when Draupnir fails to start up.
+#  # - "RecoveryOnly" will only start the bot in safe mode when there are recovery options available.
+#  # - "Never" will never start the bot in safe mode when Draupnir fails to start normally.
+#  # - "Always" will always start the bot in safe mode when Draupnir fails to start normally.
+#  bootOption: RecoveryOnly
```

Comment:
@FSG-Cat @MTRNord If either of you have opinions about this?

Reply:
This Feline approves

Reply:
Seems good with me. It would just be nice to know which possible recover options one could encounter. (as in documentation). Always nice to know what to expect up front for that.

Reply:
Oki, i'll update that